resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DepsTestsSuite.scala

```diff
@@ -125,16 +125,18 @@ private[spark] trait DepsTestsSuite { k8sSuite: KubernetesSuite =>
       .endSpec()
       .build()
 
-    kubernetesTestComponents
+    // try until the service from a previous test is deleted
+    Eventually.eventually(TIMEOUT, INTERVAL) (kubernetesTestComponents
```

Comment:
Is this relevant to this PR? If this aims to the flakiness of the existing test suite, we should make another PR before this one.

Reply:
Nope this is to address waiting for the elimination of the running service from a previous test. It is an easy retry until the previous service is removed and the new can succeed.